frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -0,0 +1,330 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import * as classNames from 'classnames';
+import { connect } from 'react-redux';
+import {
+  sortable,
+  SortByDirection,
+  ISortBy,
+  Table,
+  TableHeader,
+  TableBody,
+  TableVariant,
+} from '@patternfly/react-table';
+import { tableFilters } from '@console/internal/components/factory/table-filters';
+import { ActionGroup, Button } from '@patternfly/react-core';
+import { ButtonBar } from '@console/internal/components/utils/button-bar';
+import { history } from '@console/internal/components/utils/router';
+import {
+  SelectorInput,
+  humanizeCpuCores,
+  humanizeBinaryBytes,
+  ResourceLink,
+} from '@console/internal/components/utils/index';
+import {
+  getNodeRoles,
+  k8sCreate,
+  k8sPatch,
+  NodeKind,
+  referenceForModel,
+} from '@console/internal/module/k8s';
+import { NodeModel } from '@console/internal/models';
+import { Tooltip } from '@console/internal/components/utils/tooltip';
+import { OCSServiceModel } from '../../models';
+import { minSelectedNode, labelObj, ocsRequestData, taintObj } from '../../constants/ocs-install';
```

Comment:
`labelObj`, `ocsRequestData`, and `taintObj` are so tightly coupled to this component, I feel like they are better in this file. I sort of disagree with the earlier comment.

Reply:
@spadgett The intention was to keep all the constants related to ocs install view separately as it was making the file very lengthy. There are so many constants being used here.